docs/src/reference/expressions.md

```diff
@@ -0,0 +1,299 @@
+
+* Side-effect free
+* Simple data model with JSON-like object types: numbers, dates, strings, lists, and contexts
+* Simple syntax designed for business professionals and developers
+* Three-valued logic (true, false, null)
+
+Zeebe integrates the [Feel-Scala](https://github.com/camunda/feel-scala) engine to evaluate FEEL expressions. The following sections cover common use cases in Zeebe. A complete list of supported expressions can be found in the project's [documentation](https://camunda.github.io/feel-scala).
+
+### Access Variables
+
+A variable can be accessed by its name.
+
+```feel
+owner
+// "Paul"
+
+totalPrice
+// 21.2
+
+items
+// ["item-1", "item-2", "item-3"]
+```
+
+If a variable is a JSON document/object then it is handled as a FEEL context. A property of the context (aka nested variable property) can be accessed by `.` (a dot) and the property name.
+
+```feel
+order.id
+// "order-123"
+
+order.customer.name
+// "Paul"
+```
+
+### Boolean Expressions
+
+Values can be compared using the following operators:
+
+<table style="width:100%">
+  <tr>
+    <th>Operator</th>
+    <th>Description</th>
+    <th>Example</th>
+  </tr>
+
+  <tr>
+    <td>= (only <b>one</b> equal sign)</td>
+    <td>equal to</td>
+    <td>owner = "Paul"</td>
+  </tr>
+
+  <tr>
+    <td>!=</td>
+    <td>not equal to</td>
+    <td>owner != "Paul"</td>
+  </tr>
+
+  <tr>
+    <td>&#60;</td>
+    <td>less than</td>
+    <td>totalPrice &#60; 25</td>
+  </tr>
+
+  <tr>
+    <td>&#60;=</td>
+    <td>less than or equal to</td>
+    <td>totalPrice &#60;= 25</td>
+  </tr>
+
+  <tr>
+    <td>&#62;</td>
+    <td>greater than</td>
+    <td>totalPrice &#62; 25</td>
+  </tr>
+
+  <tr>
+    <td>&#62;=</td>
+    <td>greater than or equal to</td>
+    <td>totalPrice &#62;= 25</td>
+  </tr>
+
+   <tr>
+    <td>between _ and _</td>
+    <td>same as <i>(x &#62;= _ and x &#60;= _)</i></td>
+    <td>totalPrice between 10 and 25</td>
+   </tr>
+
+</table>
+
+Multiple boolean values can be combined as disjunction (`and`) or conjunction (`or`).
+
+```feel
+orderCount >= 5 and orderCount < 15
+
+orderCount > 15 or totalPrice > 50
+```
+
+If a variable or a nested property can be `null` then it can be compared to the `null` value. Comparing `null` to a value different from `null` results in `false`.
+
+```feel
+order = null
+// true if order is null
+
+totalCount > 5
+// false is totalCount is null
+```
+
+### String Expressions
+
+A string value must be enclosed in double quotes. Multiple string values can be concatenated using the `+` operator.
+
+```feel
+"foo" + "bar"
+// "foobar"
+```
+
+Any value can be transformed into a string value using the `string()` function.
+
+```feel
+"order-" + string(orderId)
+// "order-123"
+```
+
+More functions for string values are available as [built-in functions](https://camunda.github.io/feel-scala/feel-built-in-functions#string-functions) (e.g. contains, matches, etc.).
```

Comment:
Do we have a way to check that these external links are still working? They might break without us knowing.

Reply:
Yes. The links are checks in the CI build process.

Reply:
That's awesome 👍